### PR TITLE
Changelog 2018.09

### PR DESCRIPTION
**Pull request by tobiasd**
The CHANGELOG for the 2018.09 release. 

This should be the last PR merged before the release happens.

### CODE REVIEW

**Comment by tobiasd** [comment]
How can changing the CHANGELOG file alter the test coverage @nupplaphil? And shouldn't that report be included in the checks box and not as a comment?

**Comment by MrPetovan** [comment]
I thought CodeCov comments had been disabled in the `develop` branch too.

**Comment by nupplaphil** [comment]
I'm sorry, I thought it's disabled too because of
https://github.com/friendica/friendica/blob/01451f788f8ba0fc4f504ec435191e49a0ce912c/.codecov.yml#L4

The strange thing is that this comment doesn't appear at any other PR except this one.

It seems like it's a bug in codecov (and the increasing too, because it shows a complete different change =>https://codecov.io/gh/friendica/friendica/pull/5685/changes ).

**Comment by tobiasd** [comment]
> Did we decide to use British English rather than American English in
> the change log?

Mmm... I think the answer is yes, with a grain of German.

Could be that I used a different editor, that had a different default dictionary selected. If you like, I can take the other dictionary for the spellcheck.

**Comment by annando** [comment]
We could use Australian English ;-)

**Comment by MrPetovan** [comment]
No worries, mate!

File: CHANGELOG [hunk 1]
```diff
@@ -32,13 +32,13 @@ Version 2018.09 (2018-xx-xx) [v2018-09-19]
     Fixed a bug in the daemon mode of the background worker [annando]
     Fixed a bug in the frio theme that contact filtering [rabuzarus]
     Fixed a bug that mangled the display of some additional smileys [abanink]
-    Fixed a bug in generating registration mails [MrPetovan]
+    Fixed a bug in generating registration mails [MrPetovan
```
Comment (MrPetovan) [inline CHANGELOG]: Please restore the square bracket.

File: CHANGELOG [hunk 2]
```diff
@@ -32,13 +32,13 @@ Version 2018.09 (2018-xx-xx) [v2018-09-19]
...
     Fixed a bug that mangled the display of some additional smileys [abanink]
-    Fixed a bug in generating registration mails [MrPetovan]
+    Fixed a bug in generating registration mails [MrPetovan 
     Fixed a bug that caused blank re-share bodies [MrPetovon]
     Fixed a bug in the API handling of private mails [fabrixxm]
     Fixed a bug when calling the mail() function [miqrogroove]
     Fixed a bug that caused deleted accounts being displayed in the local directory [miqrogroove]
     Fixed a bug when checking the domain of an email address [VVelox]
-    Fixed a bug that prevented re-shares from Twitter to be shown as this [annando]
+    Fixed a bug that prevented re-shares from Twitter to be shown as this [annando
```
Comment (MrPetovan) [inline CHANGELOG]: Please restore the square bracket.

**Comment by MrPetovan** [comment]
🎉

**Pull request merged**